zest/releaser/pypi.py

```diff
@@ -252,6 +278,62 @@ def distutils_servers(self):
         # The servers all need to have a section in the config file.
         return [server for server in index_servers if self.config.has_section(server)]
 
+
+class PyprojectTomlConfig(BaseConfig):
+    """Wrapper around the pyproject.toml file if available.
+
+    This is for optional zest.releaser-specific settings::
+
+        [zest-releaser]
+        python-file-with-version = "reinout/maurits.py"
+
+
+    """
+
+    config_filename = PYPROJECTTOML_CONFIG_FILE
+
+    def __init__(self):
+        """Grab the configuration (overridable for test purposes)"""
+        # If there is a pyproject.toml in the package, parse it
+        if not os.path.exists(self.config_filename):
+            self.config = None
+            return
+        with open(self.config_filename, "rb") as tomlconfig:
+            self.config = tomllib.load(tomlconfig)
+
+    def zest_releaser_config(self):
+        if self.config is None:
+            return None
+        try:
+            result = self.config["tool"]["zest-releaser"]
+        except KeyError:
+            return None
```

Comment:
```suggestion
        except KeyError:
            logger.debug("No [tool.zest-releaser] section found in the %s", self.config_filename)
            return None
```

In the other two try/excepts of the same time, a `logger.debug` can also help to pinpoint problems when they occur with the new behaviour.

Reply:
done